ci/local/build.sh

```diff
@@ -22,7 +22,7 @@ where:
 if [[ -z "${CUDA_VISIBLE_DEVICES}" ]]; then
     NVIDIA_VISIBLE_DEVICES="all"
 else
-    NVIDIA_VISIBLE_DEVICES=${CUDA_VISIBLE_DEVICES}
+    NVIDIA_VISIBLE_DEVICES="device=${CUDA_VISIBLE_DEVICES}"
```

Comment:
I don't think this should be changed. Instead, move the "device=" string down to the GPU_OPTS variable. Otherwise this won't work for Docker < 19.

Reply:
thanks